stable/velero/values.yaml

```diff
@@ -160,6 +160,10 @@ deployRestic: false
 #        - foo
 schedules: {}
 
+# Whether or not to clean up CustomResourceDefintions when deleting a release.
+# This is useful in case there's multiple releases deployed and deleting the CRDs when deleting one would render the other broken
+cleanUpCRDs: false
```

Comment:
I'm not super knowledgeable about velero but is there any potential for data loss when all CRDs and associated CRs are deleted? If so that would be important to call out here, and we might even want to change the property to something like `dangerouslyCleanUpCRDs` in case our comments don't get persisted in end-user's values files.

Reply:
That's a good question. `Backups` can be recreated by the server via the sync controller. `BackupStorageLocations` and `VolumeSnapshotLocations` would have to be manually recreated. `Restores` are not recreated. All others are transient or can be recreated.

I'll update the comment to be more explicit about this.

Reply:
What happens if velero is installed more than once in a cluster? Is that a use case? It usually is for a Helm chart. For example, when someone has QA and prod setups running in the same cluster. In this case, if someone sets `cleanUpCRDs` to `true` for the QA setup then production will also be deleted.

Another example is when multiple teams use the same cluster. One team may not know what others are doing. If one team deletes the CRD then the CRs for any other teams using it will be deleted as well.

I would be curious to know thoughts on how this impacts velero.

Reply:
Installing Velero more than once in a cluster isn't a well-tested use case, but it's also not outside the realm of possibility. The most likely scenario I can think of for this is that individual Velero instances would be set up with specific, limited service accounts so that end users could manage specific namespaces without interfering with others. There are, however, challenges around PVs since those are cluster-scoped.

I think both of you make a good point about not exposing this parameter for most users, so I'm ok with making it CI-only. That's the main intent of this PR, though we have had users frustrated that the Helm chart didn't delete everything it made. I understand why this is the case for CRDs from Helm's perspective, so really it comes down to user education about the current design.